### ISSUE TITLE
LTS-NET: End-to-end Unsupervised Learning of Long-Term 3D Stable objects. (arXiv:2301.03426v3 [cs.CV] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#26431]
LTS-NET: End-to-end Unsupervised Learning of Long-Term 3D Stable objects. (arXiv:2301.03426v3 [cs.CV] UPDATED)<br>
https://ift.tt/Fs4zp60<br>
<p>In this research, we present an end-to-end data-driven pipeline for
determining the long-term stability status of objects within a given
environment, specifically distinguishing between static and dynamic objects.
Understanding object stability is key for mobile robots since long-term stable
objects can be exploited as landmarks for long-term localisation. Our pipeline
includes a labelling method that utilizes historical data from the environment
to generate training data for a neural network. Rather than utilizing discrete
labels, we propose the use of point-wise continuous label values, indicating
the spatio-temporal stability of individual points, to train a point cloud
regression network named LTS-NET. Our approach is evaluated on point cloud data
from two parking lots in the NCLT dataset, and the results show that our
proposed solution, outperforms direct training of a classification model for
static vs dynamic object classification.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
